scaleout/stackn/templates/celery-beat-deployment.yaml

```diff
@@ -29,7 +29,7 @@ spec:
       initContainers:
         - name: wait-for-db
           image: postgres
-          command: ['sh', '-c', 'until pg_isready --host={{ .Values.postgres.host }}.$(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace).svc.cluster.local --port={{ .Values.postgres.port }}; do echo waiting for database; sleep 2; done;']
+          command: ['sh', '-c', 'until pg_isready --host={{ include "stackn.postgres.host" . }}.$(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace).svc.cluster.local --port={{ include "stackn.postgres.port" . }}; do echo waiting for database; sleep 2; done;']
```

Comment:
Is "." at end required?

Reply:
Yes "include" requires two arguments. I don't know exactly how go templates work but the "." is special type of variable of any type, i.e it returns the complete object from "stackn.postgres.host".